### ISSUE TITLE
Tests [version 1.0]

### ISSUE THREAD

**Issue by atcher0** [gkasprow/FMC_ADC_4CHA_1G_12B#2]
###   Power Supply
* [x] Power supply
**Tested**: voltage levels + PG + LEDs

### IC
**NOTE**: I2C CLK: 100 kHz; SPI CLK: 5 MHz
- [x] EEPROM (24AA64T)
  **TESTED**: read, write
- [x] AD7291 (ADC - voltage monitor)
  **TESTED**: temperature and voltage measurements
- [x] AD9234 (ADC) 
  **TESTED**: [4/8] GT LINKS @ 10 Gbps, PRBS31; JESD204B link
- [ ] AD9528 (CLK)
  **TESTED**: PLL2 LOCK; clock and sysref generation
- [x] MAX5442 (offset correction DAC) 

### ADC analog input
- [x] Offset correction ( +/-5 V)
**NOTE:** Measured range: <- 4.83 V;  4.86 V>
- [x] Gain steps (+/-50 mV, +/-0.5 V, +/-5 V)
- [x] Linearity
- [x] FFT analysis
- [x] Bandwidth 

### Others
- [x] Trigger
- [x] LEDs

**Comment by atcher0** [comment]
f_in = 10 MHz

![adc_linearity_10MHz_50mV](https://user-images.githubusercontent.com/23083203/72226254-53c16480-358f-11ea-8264-e8a186561209.png)
![adc_linearity_10MHz_0 5V](https://user-images.githubusercontent.com/23083203/72226248-3db3a400-358f-11ea-99a2-04fa101c22c4.png)
![adc_linearity_10MHz_5V](https://user-images.githubusercontent.com/23083203/72226257-6340ad80-358f-11ea-94a6-f976f0db1a73.png)

**Comment by atcher0** [comment]
![fft_10MHz_0 5V_32000_3](https://user-images.githubusercontent.com/23083203/72226755-d4cf2a80-3594-11ea-952c-30cbc9478357.png)
![fft_10MHz_5V_32000_3](https://user-images.githubusercontent.com/23083203/72226757-d8fb4800-3594-11ea-88a2-fe456535014c.png)
![fft_10MHz_50mV_32000_3](https://user-images.githubusercontent.com/23083203/72226758-dbf63880-3594-11ea-83ee-29358417162d.png)

**Comment by gkasprow** [comment]
What generator did you use? Is it DDS? It may have a lot of harmonics.

**Comment by atcher0** [comment]
Keysight 33621A

**Comment by gkasprow** [comment]
Yes. I have a much better one at the WUT, SMB100A which has much lower phase noise and harmonics. It's also good to connect LPF before to reduce them. Bring the stuff to WUT for characterisation

**Comment by atcher0** [comment]
Yes, SMB100A is much better. I will come on Thursday.

**Comment by atcher0** [comment]
I repeated my measurements using SMB100A generator and LP (SLP-15+; DC to 15 MHz) filter.
Here are the results:
![fft_ch3_10MHz_500mV_0 85Vpp_32000](https://user-images.githubusercontent.com/23083203/73128594-12d74000-3fd2-11ea-8d59-69c6da0f7eee.png)
![fft_ch3_10MHz_5000mV_8 5Vpp_32000_2](https://user-images.githubusercontent.com/23083203/73128602-2f737800-3fd2-11ea-81e0-2d309078dd60.png)
![fft_ch3_10MHz_50mV_0 085Vpp_32000](https://user-images.githubusercontent.com/23083203/73128604-3306ff00-3fd2-11ea-8a30-7069f0f4312a.png)

**Comment by atcher0** [comment]
![bandwidth_ch3_5000mV_8 5Vpp_4000](https://user-images.githubusercontent.com/23083203/73128651-c3454400-3fd2-11ea-95dd-2576ceebbb79.png)
![bandwidth_ch3_500mV_0 5Vpp_4000](https://user-images.githubusercontent.com/23083203/73128652-c7716180-3fd2-11ea-8f7a-40cba4c4bc53.png)
![bandwidth_ch3_50mV_0 085Vpp_4000](https://user-images.githubusercontent.com/23083203/73128653-cb04e880-3fd2-11ea-8d1c-72e23d5566cc.png)

**Comment by gkasprow** [comment]
It looks much better now :)

**Comment by gkasprow** [comment]
Add testing of the external clock input

**Comment by gkasprow** [comment]
You can use the second SMB100A generator. 
Also, test the trigger IO.
